src/uct/cuda/gdr_copy/gdr_copy_md.c

```diff
@@ -37,6 +37,7 @@ static ucs_config_field_t uct_gdr_copy_md_config_table[] = {
 static ucs_status_t
 uct_gdr_copy_md_query(uct_md_h md, uct_md_attr_v2_t *md_attr)
 {
+    uct_md_base_md_query(md_attr);
```

Comment:
all the "*mem_types" fields can also be set to 0 by default

Reply:
ok, will be fixed